src/Yaapii.Atoms/Text/Strict.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Yaapii.Atoms.Enumerable;
+
+namespace Yaapii.Atoms.Text
+{
+    public sealed class Strict : TextEnvelope
```

Comment:
please add header docs also aboce the class.

Reason: if you jump through the class in Intellisense when you want to see, what exists (but yyou do not yet have a variable defined), the header docs from the classes will be displayed.

The header docs of the ctor are displayed, if you have a variable defined and hover the definition.

So we always need both.